code/modules/projectiles/projectile/bullets/pistol.dm

```diff
@@ -9,12 +9,17 @@
 	damage = 15
 	armour_penetration = 40
 
+/obj/item/projectile/bullet/c9mm_hp
+	name = "9mm hollow-point bullet"
+	damage = 25
+	armour_penetration = -50
+
 /obj/item/projectile/bullet/incendiary/c9mm
 	name = "9mm incendiary bullet"
 	damage = 10
 	fire_stacks = 1
 
-// 10mm (Stechkin)
+// 10mm (M92)

```

Comment:
Stechkin man good, etc etc.

Reply:
![image](https://user-images.githubusercontent.com/22850904/63908910-37031500-ca64-11e9-82ef-1d188f727abc.png)
 in this pr changelog